Updated FSSHTTPWOPIInspector project

IamWenboZhang opened the pull request:
1.Updated HTTP structure as the change of protocol
2.Updated function of  ParseHTTPPayloadForWOPI
3.Updated the definition of st_wopiactionvalues
4.Updated FSSHTTPInspector project to .Net framework 4.6.1

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/OfficeDev/Office-Inspectors-for-Fiddler?pullRequest=32) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/OfficeDev/Office-Inspectors-for-Fiddler?pullRequest=32) before we can accept your contribution.<br/><br/>:x: IamWenboZhang [sign now](https://cla.opensource.microsoft.com/OfficeDev/Office-Inspectors-for-Fiddler?pullRequest=32)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/OfficeDev/Office-Inspectors-for-Fiddler?pullRequest=32) it.</sub>

IamWenboZhang commented:
FSSHTTPandWOPIInspector
1.Updated HTTP structure as the change of protocol
2.Updated function of ParseHTTPPayloadForWOPI
3.Updated the definition of st_wopiactionvalues
4.Updated FSSHTTPandWOPIInspector project to .Net framework 4.6.1

WOPIautomation
1.Update FSSHTTPandWOPIInspector.dll to the last version
2.Update the reference of fiddler in WOPIautomation to fiddler4
3.Add a new reference of AutoITX3Lib
4.Changing some code to suit new SharePoint workflow
5.Update WOPIautomation project to .Net framework 4.6.1
